### DISCUSSION TITLE
E3V2-BLTouch-3x3-HS-v4.2.2.bin - Pins for the BL Touch?

### DISCUSSION THREAD

**Discussion by sadepa1337** [Jyers/Marlin#800] [General Discussion]
Greetings
I'm pretty new to the thing so I probably did a blunder.

I compiled the bugfix version myself, it works! (even if I flash back to the old version I made). Then I wanted to take advantage of the great offer and install the already compiled firmware E3V2-BLTouch-3x3-HS-v4.2.2.bin, everything worked without any problems. I just can't home anymore. X and Y axis works fine  but the BL Touch doesn't seem to work anymore. It hase power, is also triese to probe, but the Z axis does not go down a bit and the process is canceled. Did I describe it in an understandable way?
I think the pin assignment doesn't match anymore. I used the following tutorial. https://ibb.co/pZ5RywX

**Comment by smooreace** [comment]
I dont believe its supposed to be using the Z stop pins. It should be using
the proper BLTouch pins with nothing connected to the Z stop.

**Reply by sadepa1337** [reply]
Thank you for your prompt reply. That's what teaching tech guy said. What are they exactly? Can you draw them in or make a diagram?

https://i2.wp.com/smith3d.com/wp-content/uploads/2020/07/5.jpg?resize=980%2C552&ssl=1

lol? just next to the others??

**Reply by smooreace** [reply]
This might be a little more informational
![image](https://user-images.githubusercontent.com/35540547/118008264-83fb4d80-b312-11eb-9e9f-aa83edd42aa1.png)

**Reply by sadepa1337** [reply]
I thank you from the bottom of my heart, you have helped me a lot!

**Comment by isochromanone** [comment]
Just to continue this line of conversation, does the jyers pre-compiled BLTouch firmware allow for the Creality Z-stop sensor as a secondary stop?   I have a small issue (perhaps a sticky BL Touch probe) where the head will crash into the bed about 1 in 100 times when homing.  I'd like to reinstall the Z-stop switch as a "over-limit" cutoff perhaps 0.5 mm below bed (the head flexes up a few mm when crashing so this would be a good safety).
